plugin/Model/FormModelBuilder/FormModelBuilderDBForm.php

```diff
@@ -272,7 +272,7 @@ public function createInputModelRecipient()
 			(InputModelDBFactoryConfigForm::INPUT_FORM_RECIPIENT, $labelRecipient);
 		$pInputModelFormRecipient->setHtmlType(InputModelOption::HTML_TYPE_TEXT);
 		$pInputModelFormRecipient->setValue($selectedRecipient);
-		$pInputModelFormRecipient->setPlaceholder(__('john.doe@example.com', 'onoffice-for-wp-websites'));
+		$pInputModelFormRecipient->setPlaceholder(__('', 'onoffice-for-wp-websites'));
```

Comment:
This entire line is a no-op and can be removed

Reply:
I will update it